controller/iteration.go

```diff
@@ -96,9 +96,6 @@ func (c *IterationController) CreateChild(ctx *app.CreateChildIterationContext)
 			EndAt:       reqIter.Attributes.EndAt,
 			UserActive:  *reqIter.Attributes.UserActive,
 		}
-		if reqIter.ID != nil {
-			newItr.ID = *reqIter.ID
-		}
```

Comment:
A regression test would be nice here.

Reply:
added a test here in https://github.com/fabric8-services/fabric8-wit/pull/1737/commits/66c557d0d0ad5e0d5a228637b76b87d154d15440